Source/Particles/Collision/BinaryCollision/Coulomb/PairWiseCoulombCollisionFunc.H

```diff
@@ -84,7 +85,7 @@ public:
             index_type const I2s, index_type const I2e,
             index_type const* AMREX_RESTRICT I1,
             index_type const* AMREX_RESTRICT I2,
-            ParticleTileType ptile1, ParticleTileType ptile2,
```

Comment:
This was the issue in the old code. `ParticleTileType` compared to `ParticleTileDataType` does copy indeed.